src/components/Certify/Certify.vue

```diff
@@ -1,21 +1,28 @@
 <template>
   <v-card flat id="AR-step-4-container">
     <div class="pl-8 pt-8 pr-8">
-      <v-layout row>
-        <v-flex xs3 md2>
-          <label><strong>Legal Name</strong></label>
-        </v-flex>
-        <v-flex xs9 md10>
-          <div class="value certified-by">
-            <v-text-field
-              filled
-              persistent-hint
-              id="certified-by-textfield"
-              label="Legal name of authorized person"
-              :value="certifiedBy"
-              :rules="[(v) => !!v || 'A person\'s legal name is required.']"
-              @input="emitCertifiedBy($event)"
-            />
+      <v-container>
+        <v-row class="pl-4">
+          <v-col cols="3" md="3" class="firstCol">
+            <label><strong>Legal Name</strong></label>
+          </v-col>
+          <v-col cols="9" md="9" class="secondCol">
```

Comment:
I'm not sure if you need `md="9"` since it's the same as the cols value. Please remove if not needed.

Reply:
removed